# Andreas-Johann Ø Ulvestad <aj@aju.no>, 2017. #zanata
# Petr Lautrbach <plautrba@fedoraproject.org>, 2017. #zanata
msgid ""
msgstr ""
"Project-Id-Version: PACKAGE VERSION\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2017-10-31 20:57+0100\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"PO-Revision-Date: 2017-08-23 04:28-0400\n"
"Last-Translator: Petr Lautrbach <plautrba@fedoraproject.org>\n"
"Language-Team: Norwegian Nynorsk\n"
"Language: nn\n"
"X-Generator: Zanata 3.9.6\n"
"Plural-Forms: nplurals=2; plural=(n != 1)\n"

#: ../setroubleshoot.desktop.in.h:1
msgid "SELinux Troubleshooter"
msgstr ""

#: ../setroubleshoot.desktop.in.h:2
msgid "Troubleshoot SELinux access denials"
msgstr ""

#: ../setroubleshoot.desktop.in.h:3
msgid "policy;security;selinux;avc;permission;mac;alert;sealert;"
msgstr ""

#: ../src/setroubleshoot/audit_data.py:912
#, python-format
msgid "port %s"
msgstr "port %s"

#: ../src/setroubleshoot/audit_data.py:914 ../src/setroubleshoot/browser.py:759
#: ../src/setroubleshoot/browser.py:834 ../src/setroubleshoot/rpc.py:528
#: ../src/setroubleshoot/util.py:242
msgid "Unknown"
msgstr "Ukjend"

#: ../src/setroubleshoot/audit_data.py:1004
#, python-format
msgid "%s \n"
"**** Invalid AVC allowed in current policy ***\n"
msgstr "%s\n"
"**** Ugyldig AVC godkjend i noverande regelsett ***\n"

#: ../src/setroubleshoot/audit_data.py:1006
#, python-format
msgid ""
"%s \n"
"**** Invalid AVC dontaudited in current policy.  'semodule -B' will turn on "
"dontaudit rules. ***\n"
msgstr ""
"%s\n"
"**** Ugyldig AVC dontaudited i noverande regelsett. 'semodule -B' vil skru "
"på dontaudit-reglar. ***\n"

#: ../src/setroubleshoot/audit_data.py:1008
msgid "Must call policy_init first"
msgstr ""

#: ../src/setroubleshoot/audit_data.py:1010
#, python-format
msgid "%s \n"
"**** Invalid AVC bad target context. ***\n"
msgstr "%s\n"
"**** Ugyldig AVC feil kjeldekontekst. ***\n"

#: ../src/setroubleshoot/audit_data.py:1012
#, python-format
msgid "%s \n"
"**** Invalid AVC bad source context. ***\n"
msgstr "%s\n"
"**** Ugyldig AVC feil typeklasse ***\n"

#: ../src/setroubleshoot/audit_data.py:1014
#, python-format
msgid "%s \n"
"**** Invalid AVC bad type class ***\n"
msgstr "%s\n"
"**** Ugyldig AVC feil typeklasse ***\n"

#: ../src/setroubleshoot/audit_data.py:1016
#, python-format
msgid "%s \n"
"**** Invalid AVC bad permission ***\n"
msgstr ""

#: ../src/setroubleshoot/audit_data.py:1018
msgid "Error during access vector computation"
msgstr ""

#: ../src/setroubleshoot/browser.py:183 ../gui/browser.glade.h:14
msgid "SELinux Alert Browser"
msgstr ""

#: ../src/setroubleshoot/browser.py:187 ../gui/browser.glade.h:21
msgid "The source process:"
msgstr ""

#: ../src/setroubleshoot/browser.py:192 ../gui/browser.glade.h:17
msgid "Yes"
msgstr "Ja"

#: ../src/setroubleshoot/browser.py:194 ../gui/browser.glade.h:19
msgid "No"
msgstr "Nei"

#: ../src/setroubleshoot/browser.py:199 ../gui/browser.glade.h:22
msgid "Attempted this access:"
msgstr "Prøvde denne tilgangen:"

#: ../src/setroubleshoot/browser.py:213 ../gui/browser.glade.h:12
msgid "SETroubleshoot Details Window"
msgstr "SETroubleshoot detaljvindauge"

#: ../src/setroubleshoot/browser.py:216 ../gui/browser.glade.h:16
msgid "Would you like to receive alerts?"
msgstr "Vil du motta varslar?"

#: ../src/setroubleshoot/browser.py:220 ../gui/browser.glade.h:26
msgid "Notify Admin"
msgstr "Varsle Admin"

#: ../src/setroubleshoot/browser.py:223 ../src/setroubleshoot/browser.py:229
#: ../gui/browser.glade.h:8
msgid "Troubleshoot"
msgstr "Feilsøking"

#: ../src/setroubleshoot/browser.py:226 ../gui/browser.glade.h:28
msgid "Details"
msgstr "Detaljar"

#: ../src/setroubleshoot/browser.py:233 ../gui/browser.glade.h:7
msgid "SETroubleshoot Alert List"
msgstr "SETroubleshoot varselliste"

#: ../src/setroubleshoot/browser.py:235 ../gui/browser.glade.h:37
msgid "List All Alerts"
msgstr "Liste alle verslar"

#: ../src/setroubleshoot/browser.py:282
msgid "#"
msgstr "#"

#: ../src/setroubleshoot/browser.py:282
msgid "Source Process"
msgstr "Kjeldeprosess"

#: ../src/setroubleshoot/browser.py:282
msgid "Attempted Access"
msgstr "Prøvde tilgang"

#: ../src/setroubleshoot/browser.py:282
msgid "On this"
msgstr "På denne"

#: ../src/setroubleshoot/browser.py:282
msgid "Occurred"
msgstr "Skjedde"

#: ../src/setroubleshoot/browser.py:282
msgid "Status"
msgstr "Status"

#: ../src/setroubleshoot/browser.py:356 ../src/setroubleshoot/browser.py:798
#: ../src/setroubleshoot/browser.py:842
msgid "Notify"
msgstr "Varsle"

#: ../src/setroubleshoot/browser.py:357
msgid "Notify alert in the future."
msgstr "Varsle varslar i framtida."

#: ../src/setroubleshoot/browser.py:360 ../src/setroubleshoot/browser.py:800
#: ../src/setroubleshoot/browser.py:840 ../gui/browser.glade.h:29
msgid "Ignore"
msgstr "Ignorer"

#: ../src/setroubleshoot/browser.py:361 ../gui/browser.glade.h:30
msgid "Ignore alert in the future."
msgstr "Ignorer varslar i framtida."

#: ../src/setroubleshoot/browser.py:397
msgid "<b>If you were trying to...</b>"
msgstr "<b>Dersom du prøvde å...</b>"

#: ../src/setroubleshoot/browser.py:406
msgid "<b>Then this is the solution.</b>"
msgstr "<b>Så er dette løysninga.</b>"

#: ../src/setroubleshoot/browser.py:513
msgid "Plugin\n"
"Details"
msgstr ""

#: ../src/setroubleshoot/browser.py:528
msgid "Report\n"
"Bug"
msgstr "Rapport\n"
"Feil"

#: ../src/setroubleshoot/browser.py:556
#, python-format
msgid "Plugin: %s "
msgstr "Programtillegg: %s"

#: ../src/setroubleshoot/browser.py:623
msgid "Unable to grant access."
msgstr "Kunne ikkje gje tilgang."

#: ../src/setroubleshoot/browser.py:654 ../src/setroubleshoot/browser.py:796
#, python-format
msgid "Alert %d of %d"
msgstr "Varsel %d av %d"

#: ../src/setroubleshoot/browser.py:779
#, python-format
msgid "On this %s:"
msgstr "På denne %s:"

#: ../src/setroubleshoot/browser.py:827 ../src/setroubleshoot/browser.py:833
#: ../src/setroubleshoot/signature.py:404
msgid "N/A"
msgstr "I/G"

#: ../src/setroubleshoot/browser.py:874 ../src/setroubleshoot/browser.py:875
#: ../gui/browser.glade.h:34
msgid "No Alerts"
msgstr "Inga varslar"

#: ../src/setroubleshoot/browser.py:893
msgid "SELinux has detected a problem."
msgstr "SELinux har merka eit problem."

#: ../src/setroubleshoot/browser.py:999
msgid "Sealert Error"
msgstr "Sealert-feil"

#: ../src/setroubleshoot/browser.py:1010
msgid "Sealert Message"
msgstr "Sealert-melding"

#. -----------------------------------------------------------------------------
#: ../src/setroubleshoot/errcode.py:79
msgid "signature not found"
msgstr "signatur var ikkje funne"

#: ../src/setroubleshoot/errcode.py:80
msgid "multiple signatures matched"
msgstr "fleire signaturar stemte"

#: ../src/setroubleshoot/errcode.py:81
msgid "id not found"
msgstr "id ikkje funne"

#: ../src/setroubleshoot/errcode.py:82
msgid "database not found"
msgstr "database ikkje funne"

#: ../src/setroubleshoot/errcode.py:83
msgid "item is not a member"
msgstr "ting er ikkje eit medlem"

#: ../src/setroubleshoot/errcode.py:84
msgid "illegal to change user"
msgstr "ulovleg å endre brukar"

#: ../src/setroubleshoot/errcode.py:85
msgid "method not found"
msgstr "metode ikkje funne"

#: ../src/setroubleshoot/errcode.py:86
msgid "cannot create GUI"
msgstr "kunne ikkje opprette GUI"

#: ../src/setroubleshoot/errcode.py:87
msgid "value unknown"
msgstr "ukjend verdi"

#: ../src/setroubleshoot/errcode.py:88
msgid "cannot open file"
msgstr "kan ikkje opne fil"

#: ../src/setroubleshoot/errcode.py:89
msgid "invalid email address"
msgstr "ugyldig epostadresse"

#. gobject IO Errors
#: ../src/setroubleshoot/errcode.py:92
msgid "socket error"
msgstr ""

#: ../src/setroubleshoot/errcode.py:93
msgid "connection has been broken"
msgstr "kopling vart broten"

#: ../src/setroubleshoot/errcode.py:94
msgid "Invalid request. The file descriptor is not open"
msgstr "Ugyldig førespurnad. Filpeikaren er ikkje open"

#: ../src/setroubleshoot/errcode.py:96
msgid "insufficient permission to modify user"
msgstr "ikkje nok tilgang til å endre brukar"

#: ../src/setroubleshoot/errcode.py:97
msgid "authentication failed"
msgstr "autentisering feila"

#: ../src/setroubleshoot/errcode.py:98
msgid "user prohibited"
msgstr "brukar nekta"

#: ../src/setroubleshoot/errcode.py:99
msgid "not authenticated"
msgstr "ikkje autentisert"

#: ../src/setroubleshoot/errcode.py:100
msgid "user lookup failed"
msgstr "brukaroppslag feila"

#: ../src/setroubleshoot/gui_utils.py:53 ../src/sealert:539
#, c-format, python-format
msgid "Opps, %s hit an error!"
msgstr "Oops, %s traff ein feil!"

#: ../src/setroubleshoot/gui_utils.py:55 ../gui/fail_dialog.glade.h:1
msgid "Error"
msgstr "Feil"

#: ../src/setroubleshoot/Plugin.py:90
msgid ""
"If you want to allow $SOURCE_BASE_PATH to have $ACCESS access on the "
"$TARGET_BASE_PATH $TARGET_CLASS"
msgstr ""

#: ../src/setroubleshoot/server.py:214
#, python-format
msgid " For complete SELinux messages run: sealert -l %s"
msgstr ""

#: ../src/setroubleshoot/server.py:401
#, python-format
msgid "The user (%s) cannot modify data for (%s)"
msgstr "Brukaren (%s) kan ikkje endre data for (%s)"

#: ../src/setroubleshoot/server.py:465 ../src/sealert:299
msgid "Started"
msgstr "Starta"

#: ../src/setroubleshoot/server.py:663
msgid "AVC"
msgstr "AVC"

#: ../src/setroubleshoot/server.py:729
msgid "Audit Listener"
msgstr "Audit Listener"

#: ../src/setroubleshoot/signature.py:80
msgid "Never Ignore"
msgstr "Aldri ignorer"

#: ../src/setroubleshoot/signature.py:81
msgid "Ignore Always"
msgstr "Ignorer alltid"

#: ../src/setroubleshoot/signature.py:82
msgid "Ignore After First Alert"
msgstr "Ignorer etter første varsel"

#: ../src/setroubleshoot/signature.py:199
msgid "directory"
msgstr "katalog"

#: ../src/setroubleshoot/signature.py:200
msgid "semaphore"
msgstr "semafor"

#: ../src/setroubleshoot/signature.py:201
msgid "shared memory"
msgstr "delt minne"

#: ../src/setroubleshoot/signature.py:202
msgid "message queue"
msgstr "meldingskø"

#: ../src/setroubleshoot/signature.py:203
msgid "message"
msgstr "melding"

#: ../src/setroubleshoot/signature.py:204
msgid "file"
msgstr "fil"

#: ../src/setroubleshoot/signature.py:205
msgid "socket"
msgstr ""

#: ../src/setroubleshoot/signature.py:206
msgid "process"
msgstr "prosess"

#: ../src/setroubleshoot/signature.py:207
msgid "process2"
msgstr ""

#: ../src/setroubleshoot/signature.py:208
msgid "filesystem"
msgstr "filsystem"

#: ../src/setroubleshoot/signature.py:209
msgid "node"
msgstr "node"

#: ../src/setroubleshoot/signature.py:210
msgid "capability"
msgstr ""

#: ../src/setroubleshoot/signature.py:211
msgid "capability2"
msgstr ""

#: ../src/setroubleshoot/signature.py:388
#, python-format
msgid "%s has a permissive type (%s). This access was not denied."
msgstr ""

#: ../src/setroubleshoot/signature.py:391
msgid "SELinux is in permissive mode. This access was not denied."
msgstr ""

#: ../src/setroubleshoot/signature.py:445
#, python-format
msgid "SELinux is preventing %s from using the %s access on a process."
msgstr ""

#: ../src/setroubleshoot/signature.py:445
#, python-format
msgid "SELinux is preventing %s from using the '%s' accesses on a process."
msgstr "SELinux hindrar %s frå å bruke '%s'-tilgangen på ein prosess."

#: ../src/setroubleshoot/signature.py:448
#, python-format
msgid "SELinux is preventing %s from using the %s capability."
msgstr "SELinux hindrar %s frå å bruke %s-funksjonen."

#: ../src/setroubleshoot/signature.py:448
#, python-format
msgid "SELinux is preventing %s from using the '%s' capabilities."
msgstr "SELinux hindrar %s frå å bruke '%s'-funksjonen."

#: ../src/setroubleshoot/signature.py:450
#, python-format
msgid "SELinux is preventing %s from %s access on the %s labeled %s."
msgstr ""

#: ../src/setroubleshoot/signature.py:450
#, python-format
msgid "SELinux is preventing %s from '%s' accesses on the %s labeled %s."
msgstr ""

#: ../src/setroubleshoot/signature.py:451
#, python-format
msgid "SELinux is preventing %s from %s access on the %s %s."
msgstr "SELinux hindrar %s frå å få tilgang til %s på %s %s."

#: ../src/setroubleshoot/signature.py:451
#, python-format
msgid "SELinux is preventing %s from '%s' accesses on the %s %s."
msgstr "SELinux hindrar %s frå %s frå %s sin tilgang på %s %s."

#: ../src/setroubleshoot/signature.py:483
msgid "Additional Information:\n"
msgstr "Meir informasjon:\n"

#: ../src/setroubleshoot/signature.py:484
msgid "Source Context"
msgstr "Kjeldekontekst"

#: ../src/setroubleshoot/signature.py:485
msgid "Target Context"
msgstr "Målkontekst"

#: ../src/setroubleshoot/signature.py:486
msgid "Target Objects"
msgstr "Målobjekter"

#: ../src/setroubleshoot/signature.py:487
msgid "Source"
msgstr "Kjelde"

#: ../src/setroubleshoot/signature.py:488
msgid "Source Path"
msgstr "Kjeldebane"

#: ../src/setroubleshoot/signature.py:489
msgid "Port"
msgstr "Port"

#: ../src/setroubleshoot/signature.py:491
#: ../src/setroubleshoot/signature.py:493
msgid "Host"
msgstr "Vert"

#: ../src/setroubleshoot/signature.py:494
msgid "Source RPM Packages"
msgstr "Kjelde-RPM-pakker"

#: ../src/setroubleshoot/signature.py:495
msgid "Target RPM Packages"
msgstr "Mål-RPM-pakker"

#: ../src/setroubleshoot/signature.py:496
msgid "Policy RPM"
msgstr "Regelsett-RPM"

#: ../src/setroubleshoot/signature.py:497
msgid "Selinux Enabled"
msgstr "Selinux aktivert"

#: ../src/setroubleshoot/signature.py:498
msgid "Policy Type"
msgstr "Regeltype"

#: ../src/setroubleshoot/signature.py:499
msgid "Enforcing Mode"
msgstr "Handhevingmodus"

#: ../src/setroubleshoot/signature.py:501
#: ../src/setroubleshoot/signature.py:503
msgid "Host Name"
msgstr "Vertnamn"

#: ../src/setroubleshoot/signature.py:508
#: ../src/setroubleshoot/signature.py:510
msgid "Platform"
msgstr "Plattform"

#: ../src/setroubleshoot/signature.py:511
msgid "Alert Count"
msgstr "Varseltellar"

#: ../src/setroubleshoot/signature.py:513
msgid "First Seen"
msgstr "Først sett"

#: ../src/setroubleshoot/signature.py:514
msgid "Last Seen"
msgstr "Sist sett"

#: ../src/setroubleshoot/signature.py:515
msgid "Local ID"
msgstr "Lokal ID"

#: ../src/setroubleshoot/signature.py:517
msgid "Raw Audit Messages"
msgstr ""

#: ../src/setroubleshoot/signature.py:571
#, python-format
msgid "\n"
"\n"
"*****  Plugin %s (%.4s confidence) suggests   "
msgstr ""

#: ../src/setroubleshoot/signature.py:574
msgid "*"
msgstr "*"

#: ../src/setroubleshoot/signature.py:575
#: ../src/setroubleshoot/signature.py:577
msgid "\n"
msgstr ""

#: ../src/setroubleshoot/signature.py:579
msgid "\n"
"Then "
msgstr ""

#: ../src/setroubleshoot/signature.py:582
msgid "\n"
"Do\n"
msgstr ""

#: ../src/setroubleshoot/signature.py:584
msgid "\n"
"\n"
msgstr ""

#: ../src/updater.py:18
#, python-format
msgid "current: %s "
msgstr ""

#: ../src/updater.py:27
#, python-format
msgid "newer: %s "
msgstr ""

#: ../src/seapplet.c:189
msgid "New SELinux security alert"
msgstr ""

#: ../src/seapplet.c:189
msgid "AVC denial, click icon to view"
msgstr ""

#: ../src/seapplet.c:195 ../src/seapplet.c:246
msgid "Dismiss"
msgstr ""

#: ../src/seapplet.c:205 ../src/seapplet.c:256
msgid "Show"
msgstr ""

#. set tooltip
#: ../src/seapplet.c:440 ../src/sealert:67
msgid "SELinux AVC denial, click to view"
msgstr ""

#: ../src/sealert:109
msgid "SELinux not enabled, sealert will not run on non SELinux systems"
msgstr ""

#: ../src/sealert:156
msgid "Not fixable."
msgstr ""

#: ../src/sealert:163
#, c-format
msgid "Successfully ran %s"
msgstr ""

#: ../src/sealert:170
#, c-format
msgid "Plugin %s not valid for %s id"
msgstr ""

#: ../src/setroubleshootd:73
msgid "SELinux not enabled, setroubleshootd exiting..."
msgstr ""

#: ../src/setroubleshootd:105
#, c-format
msgid "fork #1 failed: %d (%s)"
msgstr ""

#: ../gui/browser.glade.h:1
msgid ""
"Copyright (c) 2010 \n"
"Thomas Liu <tliu@redhat.com>\n"
"Máirín Duffy <duffy@redhat.com>\n"
"Daniel Walsh <dwalsh@redhat.com>\n"
"John Dennis <jdennis@redhat.com>\n"
msgstr ""

#: ../gui/browser.glade.h:9
msgid "Troubleshoot selected alert"
msgstr ""

#: ../gui/browser.glade.h:10
msgid "Delete"
msgstr ""

#: ../gui/browser.glade.h:11
msgid "Delete Selected Alerts"
msgstr ""

#: ../gui/browser.glade.h:13
msgid "Close"
msgstr ""

#: ../gui/browser.glade.h:15
msgid "<b>SELinux has detected a problem.</b>"
msgstr ""

#: ../gui/browser.glade.h:18
msgid "Turn on alert pop-ups."
msgstr ""

#: ../gui/browser.glade.h:20
msgid "Turn off alert pop-ups."
msgstr "Skru av varsel-sprettoppvindauger."

#: ../gui/browser.glade.h:23
msgid "On this file:"
msgstr "På denne fila:"

#: ../gui/browser.glade.h:24
msgid "label"
msgstr "merke"

#: ../gui/browser.glade.h:25
msgid ""
"Read alert troubleshoot information.  May require administrative privileges "
"to remedy."
msgstr ""

#: ../gui/browser.glade.h:27
msgid "Email alert to system administrator."
msgstr "Send e-post til systemadministratoren."

#: ../gui/browser.glade.h:31
msgid "Delete current alert from the database."
msgstr "Slett noverande varslar frå databasen."

#: ../gui/browser.glade.h:32
msgid "Previous"
msgstr ""

#: ../gui/browser.glade.h:33
msgid "Show previous alert."
msgstr "Vis forrige varsel."

#: ../gui/browser.glade.h:35
msgid "Next"
msgstr ""

#: ../gui/browser.glade.h:36
msgid "Show next alert."
msgstr "Vis neste varsel."

#: ../gui/browser.glade.h:38
msgid "List all alerts in the database."
msgstr "List alle varslar i databasen."

#: ../gui/bug_report.glade.h:1
msgid "Review and Submit Bug Report"
msgstr "Sjå over og send inn feilrapportar"

#: ../gui/bug_report.glade.h:2
msgid "<span size='large' weight='bold'>Review and Submit Bug Report</span>"
msgstr ""
"<span size='large' weight='bold'>Sjå over og send inn feilrapport</span>"

#: ../gui/bug_report.glade.h:3
msgid ""
"You may wish to review the error output that will be included in this bug "
"report and modify it to exclude any sensitive data below."
msgstr ""

#: ../gui/bug_report.glade.h:4
msgid "Included error output:"
msgstr ""

#: ../gui/bug_report.glade.h:5
msgid "Submit Report"
msgstr "Send inn rapport"

#: ../gui/fail_dialog.glade.h:2 ../gui/success_dialog.glade.h:2
msgid ""
"This operation was completed.  The quick brown fox jumped over the lazy dog."
msgstr ""

#: ../gui/success_dialog.glade.h:1
msgid "Success!"
msgstr "Suksess!"

#: ../gui/success_dialog.glade.h:3
msgid "button"
msgstr "knapp"
